src/components/CustomerQuota/ItemsSelection/ItemsSelectionCard.tsx

```diff
@@ -30,22 +31,47 @@ export const ItemsSelectionCard: FunctionComponent<ItemsSelectionCard> = ({
   updateCart
 }) => {
   const [isAddUserModalVisible, setIsAddUserModalVisible] = useState(false);
+  const { getProduct } = useProductContext();
+  const identifiers = cart.flatMap(
+    cartItem => getProduct(cartItem.category)?.identifiers
+  );
+
+  /* 
+  Current condition check if there is at least an identifier 
+  that exist (!undefined) in multiple categories, and removes '+add' 
+  button if that is true
+  */
   return (
     <View>
-      <CustomerCard
-        nrics={nrics}
-        onAddNric={() => setIsAddUserModalVisible(true)}
-      >
-        <View style={sharedStyles.resultWrapper}>
-          {cart.map(cartItem => (
-            <Item
-              key={cartItem.category}
-              cartItem={cartItem}
-              updateCart={updateCart}
-            />
-          ))}
-        </View>
-      </CustomerCard>
+      {!identifiers.every(identifier => !identifier) &&
```

Comment:
haha do you wanna try this instead? I think this is cleaner and have less repeat code ^^

```
      <CustomerCard
        nrics={nrics}
        onAddNric={
          !identifiers.every(identifier => !identifier) &&
          identifiers.length > 0
            ? undefined
            : () => setIsAddUserModalVisible(true)
        }
      >
        <View style={sharedStyles.resultWrapper}>
          {cart.map(cartItem => (
            <Item
              key={cartItem.category}
              cartItem={cartItem}
              updateCart={updateCart}
            />
          ))}
        </View>
      </CustomerCard>
```